// @expo-google-fonts/sarabun/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Sarabun_100Thin = require('./100Thin/Sarabun_100Thin.ttf');
export const Sarabun_100Thin_Italic = require('./100Thin_Italic/Sarabun_100Thin_Italic.ttf');
export const Sarabun_200ExtraLight = require('./200ExtraLight/Sarabun_200ExtraLight.ttf');
export const Sarabun_200ExtraLight_Italic = require('./200ExtraLight_Italic/Sarabun_200ExtraLight_Italic.ttf');
export const Sarabun_300Light = require('./300Light/Sarabun_300Light.ttf');
export const Sarabun_300Light_Italic = require('./300Light_Italic/Sarabun_300Light_Italic.ttf');
export const Sarabun_400Regular = require('./400Regular/Sarabun_400Regular.ttf');
export const Sarabun_400Regular_Italic = require('./400Regular_Italic/Sarabun_400Regular_Italic.ttf');
export const Sarabun_500Medium = require('./500Medium/Sarabun_500Medium.ttf');
export const Sarabun_500Medium_Italic = require('./500Medium_Italic/Sarabun_500Medium_Italic.ttf');
export const Sarabun_600SemiBold = require('./600SemiBold/Sarabun_600SemiBold.ttf');
export const Sarabun_600SemiBold_Italic = require('./600SemiBold_Italic/Sarabun_600SemiBold_Italic.ttf');
export const Sarabun_700Bold = require('./700Bold/Sarabun_700Bold.ttf');
export const Sarabun_700Bold_Italic = require('./700Bold_Italic/Sarabun_700Bold_Italic.ttf');
export const Sarabun_800ExtraBold = require('./800ExtraBold/Sarabun_800ExtraBold.ttf');
export const Sarabun_800ExtraBold_Italic = require('./800ExtraBold_Italic/Sarabun_800ExtraBold_Italic.ttf');
