// @expo-google-fonts/kanit/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Kanit_100Thin = require('./100Thin/Kanit_100Thin.ttf');
export const Kanit_100Thin_Italic = require('./100Thin_Italic/Kanit_100Thin_Italic.ttf');
export const Kanit_200ExtraLight = require('./200ExtraLight/Kanit_200ExtraLight.ttf');
export const Kanit_200ExtraLight_Italic = require('./200ExtraLight_Italic/Kanit_200ExtraLight_Italic.ttf');
export const Kanit_300Light = require('./300Light/Kanit_300Light.ttf');
export const Kanit_300Light_Italic = require('./300Light_Italic/Kanit_300Light_Italic.ttf');
export const Kanit_400Regular = require('./400Regular/Kanit_400Regular.ttf');
export const Kanit_400Regular_Italic = require('./400Regular_Italic/Kanit_400Regular_Italic.ttf');
export const Kanit_500Medium = require('./500Medium/Kanit_500Medium.ttf');
export const Kanit_500Medium_Italic = require('./500Medium_Italic/Kanit_500Medium_Italic.ttf');
export const Kanit_600SemiBold = require('./600SemiBold/Kanit_600SemiBold.ttf');
export const Kanit_600SemiBold_Italic = require('./600SemiBold_Italic/Kanit_600SemiBold_Italic.ttf');
export const Kanit_700Bold = require('./700Bold/Kanit_700Bold.ttf');
export const Kanit_700Bold_Italic = require('./700Bold_Italic/Kanit_700Bold_Italic.ttf');
export const Kanit_800ExtraBold = require('./800ExtraBold/Kanit_800ExtraBold.ttf');
export const Kanit_800ExtraBold_Italic = require('./800ExtraBold_Italic/Kanit_800ExtraBold_Italic.ttf');
export const Kanit_900Black = require('./900Black/Kanit_900Black.ttf');
export const Kanit_900Black_Italic = require('./900Black_Italic/Kanit_900Black_Italic.ttf');
